install.sh

```diff
@@ -1,43 +1,52 @@
+#!/bin/bash
+
+# exit on fail
+set -e
+
 #chromebrew directories
 OWNER="skycocker"
 REPO="chromebrew"
 BRANCH="master"
-URL="https://raw.githubusercontent.com/$OWNER/$REPO/$BRANCH"
-CREW_PREFIX=${CREW_PREFIX:-/usr/local}
-CREW_LIB_PATH=$CREW_PREFIX/lib/crew/
-CREW_CONFIG_PATH=$CREW_PREFIX/etc/crew/
-CREW_BREW_DIR=$CREW_PREFIX/tmp/crew/
-CREW_DEST_DIR=$CREW_BREW_DIR/dest
-CREW_PACKAGES_PATH=$CREW_LIB_PATH/packages
-
-architecture=$(uname -m)
-
-if [ $EUID -eq 0 ]; then
-  echo 'Chromebrew should not be run as root.'
+URL="https://raw.githubusercontent.com/${OWNER}/${REPO}/${BRANCH}"
+CREW_PREFIX="${CREW_PREFIX:-/usr/local}"
+CREW_LIB_PATH="${CREW_PREFIX}/lib/crew/"
+CREW_CONFIG_PATH="${CREW_PREFIX}/etc/crew/"
+CREW_BREW_DIR="${CREW_PREFIX}/tmp/crew/"
+CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
+CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
+
+ARCH="$(uname -m)"
+
+if [ "${EUID}" == "0" ]; then
+  echo 'Chromebrew should not be installed or run as root.'
   exit 1;
 fi
 
-case "$architecture" in
+case "${ARCH}" in
 "i686"|"x86_64"|"armv7l"|"aarch64")
+  LIB_SUFFIX=
+  [ "${ARCH}" == "x86_64" ] && LIB_SUFFIX='64'
   ;;
 *)
   echo 'Your device is not supported by Chromebrew yet.'
   exit 1;;
 esac
 
 # This will allow things to work without sudo
-sudo chown -R `id -u`:`id -g` "${CREW_PREFIX}"
+sudo chown -R "$(id -u)":"$(id -g)" "${CREW_PREFIX}"
```

Comment:
This is definitely a better way to get these values.

Reply:
If a user's name has spaces in it, this is most definitely better.

Reply:
Have you ever tried creating a user with spaces?  I don't think you will have much success.  Quotes are not necessary.

Reply:
It's always better to quote variables. Just trust me here.
Anyhow, quotes aren't doing any harm.

Reply:
Please see [Quoting](https://www.gnu.org/software/bash/manual/html_node/Quoting.html).